src/main/java/org/randywebb/wlts/beans/AbstractBean.java

```diff
@@ -148,10 +149,10 @@ public AbstractBean update(JSONObject definition, String[] expectedKeys) {
         See booleanFields, dateFields, integerFields, and dateFormat.
         @return Type-correct map for CSV generation.
     */
-    public Map<String,Object> forCSV() {
-        HashMap<String,Object> results = new HashMap<String,Object>();
+    public Map<String, Object> forCSV() {
+        HashMap<String, Object> results = new HashMap<String, Object>();
```

Comment:
This is actually required by the CSV module we are using. Since the "beans" are not strictly beans anymore, the other interface to use for CSV generation is Map<String,Object>. I wasn't happy with it either.

Reply:
Perfect. I'm OK since it is required by an API that we're using. This may have even been my code, I can't remember. It showed up in the diff though, so I asked :)
Sounds like we're of the same mindset WRT collections and generics vs raw Objects, but sometimes we just have to use the Object due to external constraints.